dot.files/bin/vol

```diff
@@ -1,15 +1,15 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Code below is incompatible with `sh`, so, rather than changing the code, I've just set it to use `bash`.